suite/tests/linux/sigaction.c

```diff
@@ -115,56 +123,96 @@ set_sigaction_handler(int sig, void *action)
     assert(rc == 0);
 }
 
-#if !defined(MACOS)
+static int
+make_sigprocmask(int how, void *sigset, void *oldsigset, int size)
+{
+#if defined(MACOS)
+    /* XXX: Couldn't get the raw syscall to work on Mac. */
```

Comment:
With the BSD top bit set?

Reply:
I tried it, but it didn't work natively either. "Invalid syscall" was the error IIRC.

Reply:
Unfortunately, gdb on the GA runner is showing some errors so I'm not able to figure out what the wrapper is doing.

Reply:
A lot easier if you have access to a local Mac.  Note that gdb is not supported on Mac these days and you're supposed to use lldb -- but lldb has a bug continuing past signals.  I remember trying to build my own lldb with the bug patched but failing to get it to build and getting frustrated.

Reply:
I'm keeping the wrapper for this PR. But i'll try finding what's wrong and change it in a future PR  if I find.

Reply:
Ack